pkg/k8s/network_policy_test.go

```diff
@@ -1640,10 +1640,10 @@ func (s *K8sSuite) TestGetPolicyLabelsv1(c *C) {
 		c.Assert(lbls, NotNil)
 		c.Assert(len(lbls), Equals, 4, Commentf(
 			"Incorrect number of labels: Expected Name, UID, Namespace and DerivedFrom labels."))
-		assertLabel(lbls[0], k8sConst.PolicyLabelName, tt.name)
-		assertLabel(lbls[1], k8sConst.PolicyLabelUID, tt.uuid)
+		assertLabel(lbls[0], k8sConst.PolicyLabelDerivedFrom, tt.derivedFrom)
```

Comment:
Do not use constants here, use their values instead, if we ever change the value of the constant we might change the order accidentally.

Reply:
Changed.